rest/_examples/zones.go

```diff
@@ -62,7 +62,7 @@ func main() {
 	}
 
 	// Add an A record with a single static answer.
-	orchidRec := dns.NewRecord(domain, "orchid", "A")
+	orchidRec := dns.NewRecord(domain, "orchid", "A", nil, nil)
```

Comment:
Ah - I should have put a note in the PR.
I only changed this because dns.NewRecord has taken 5 params since https://github.com/ns1/ns1-go/pull/220 some time ago and the example was already broken.
I probably should have raised a separate issue. I can move this to a different issue if you like.

Reply:
No worries. We should have made a major version bump or added a new "constructor" at that time, but we are where we are. :laughing:

Since this is just an example fix from your PR, I think we're good.